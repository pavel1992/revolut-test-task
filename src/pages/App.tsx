import React from 'react';
import { Provider } from 'react-redux'

import { AppContainer } from "../atoms/AppÐ¡ontainer";
import { MoneyExchanger } from "../organisms/MoneyExchanger";
import { store } from "../store/store";

export const App = () => (
    <Provider store={store}>
        <AppContainer>
            <MoneyExchanger/>
        </AppContainer>
    </Provider>
);
