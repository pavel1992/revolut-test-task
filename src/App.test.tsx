import * as React from 'react';
import renderer from 'react-test-renderer';

import { App } from './App';

test('renders app correctly', () => {
  const renderedApp = renderer
    .create(<App/>)
    .toJSON();
  expect(renderedApp).toMatchSnapshot();
});
