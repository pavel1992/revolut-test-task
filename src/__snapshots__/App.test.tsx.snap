// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app correctly 1`] = `
<div
  className="sc-AxjAm lkwWUj"
>
  <div>
    Couldn't get exchange rates for exchange
  </div>
</div>
`;
